src/main/update/launcher-update.ts

```diff
@@ -307,9 +192,7 @@ export async function update(update: Update, listeners: IUpdateOptions) {
     config
   );
 
-  if (playerDownloadUrl) {
-    await playerUpdate(playerDownloadUrl, win);
-  }
+  await playerUpdate(context.urls.player, win);
```

Comment:
`src/main/update/update.ts` calls playerUpdate already, making this a double call.

Reply:
Nope, `playerUpdate` is called only when `launcher update` is not called
But `launcher update` should decouple `player update` later

Reply:
And now done, not later